beacon-chain/forkchoice/doubly-linked-tree/store.go

```diff
@@ -12,6 +12,10 @@ import (
 	"go.opencensus.io/trace"
 )
 
+// liveFraction is the minimum committee vote for the previous head that we
+// require before orphaning it's child
+var liveFraction = uint64(80)
```

Comment:
liveFraction is not that informative for a package global. Either make it more descriptive or we can move it as a global config parameter